### ISSUE TITLE
Incremental Cycle doesn't work properly with Rotation Tween.

### ISSUE THREAD

**Issue by Alvarden15** [KyryloKuzyk/PrimeTween#19]
Hello there!
Not sure if anyone else has this issue, too, but for the LocalRotation tween, when i use the Incremental Cycle Mode, the cycles don't work properly. It loops only once, then it rewinds like it was set to Yoyo Cycle Mode, and then it stops.

This only happens to me with any Tween that affects Rotation. It works fine with other tweens like Position.

Here is a sample of the code i used.

`Tween.LocalRotation(rotateTransform, new Vector3(0, 0, 180), 0.3f, Ease.Linear, -1, CycleMode.Incremental, useUnscaledTime: true);`

Thank you for your time.

**Comment by KyryloKuzyk** [comment]
PrimeTween's rotation animations use Quaternion.Slerp() to interpolate values. This method interpolates between two rotations using the shortest path. But when the total rotation angle is 180 degrees, the rotation direction is undefined because there is no one shortest path (the number of possible rotations is actually infinite). So in the case of interpolating from 0 to 180, Unity selects one direction, but when the tween increments, the rotation happens in the opposite direction from 360 back to 180. This rotation direction is not actually 'undefined', there is some logic to it, but from the user's perspective, we should treat the rotation direction as unpredictable in the case of 180-degree rotations.

There are a couple of ways to address this problem. One solution is to use Tween.EulerAngles/LocalEulerAngles(). The only major difference is that you have to specify the startValue:
```csharp
Tween.LocalEulerAngles(rotateTransform, startValue: Vector3.zero, new Vector3(0, 0, 180), 0.3f, Ease.Linear, -1, CycleMode.Incremental, useUnscaledTime: true);
```

Another solution is to rotate only by 90 degrees at a time and make the duration twice shorter:
```csharp
Tween.LocalRotation(rotateTransform, new Vector3(0, 0, 90), 0.15f, Ease.Linear, -1, CycleMode.Incremental, useUnscaledTime: true);
```

**Comment by KyryloKuzyk** [comment]
I fixed the bug in version **1.0.16**. Could you please update and tell me if it fixes the issue for you?

**Comment by Alvarden15** [comment]
> I fixed the bug in version **1.0.16**. Could you please update and tell me if it fixes the issue for you?

It's fixed now. Thank you!
